controllers/apply/awards-for-all/fields.js

```diff
@@ -347,13 +343,9 @@ module.exports = function fieldsFor({ locale, data = {}, flags = {} }) {
                 autocomplete: 'postal-code'
             },
             isRequired: true,
-            schema: Joi.when('projectCountry', {
```

Comment:
As discussed removed these on the basis that `projectCountry` is always required so I don't think there's logically a case where we'd need to strip anything here, or that joi ever even runs it, so should be safe to go. Happy to remove this change if we'd rather just focus this change on test updates.